### PR TITLE
Fix normalization

### LINKED ISSUE #1315: If table name is a java keyword, querydsl-maven-plugin generates invalid java.
For example, if a table is named "class", QClass.java will have the following line:

public static final QClass class = new QClass("class");

which is obviously illegal.

### PR DESCRIPTION
**Pull request by timowest**
Fixes #1315

### CODE REVIEW

File: querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/AbstractNamingStrategy.java [hunk 1]
```diff
@@ -77,31 +77,31 @@ public String getPrimaryKeysVariable(EntityType entityType) {
...
     @Override
     public String normalizeSchemaName(String schemaName) {
-        if (schemaName != null) {
-            return schemaName.replaceAll("\r", "").replaceAll("\n", " ");
+        return normalizeSQLName(schemaName);
+    }
+
+    protected String normalizeSQLName(String name) {
+        if (name != null) {
+            return name.replaceAll("ŗ", "").replaceAll("ņ", " ");
```
Comment (Shredder121) [inline querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/AbstractNamingStrategy.java]: This drops all `r`s and collapses all `n`s to ` `.

**Comment by timowest** [comment]
@Shredder121 @johnktims Is this good to merge?

**Comment by johnktims** [comment]
It looks good to me but I think @Shredder121 should take a look before you merge it.

**Comment by Shredder121** [comment]
Yes looks good, I'll merge this one for a change.

**Pull request merged**